lib/utils/device.js

```diff
@@ -2,6 +2,8 @@ import { Dimensions, Platform } from 'react-native'
 
 const IPHONE_X_WIDTH = 375
 const IPHONE_X_HEIGHT = 812
+const IPHONE_XR_XSMAX_WIDTH = 414;
```

Comment:
Could you please remove the semicolons in line 5 and 6? This way we have a consistent code style everywhere.